geode-core/src/main/java/org/apache/geode/management/internal/beans/GatewaySenderMBean.java

```diff
@@ -214,11 +214,6 @@ public boolean isParallel() {
     return bridge.isParallel();
   }
 
-  @Override
-  public boolean mustGroupTransactionEvents() {
-    return bridge.mustGroupTransactionEvents();
-  }
```

Comment:
why you remove these parts?

Reply:
these are mBean methods, he is not using jmx to turn off the flags on the servers, he is using function calls now.